src/core/chart-container.tsx

```diff
@@ -89,8 +91,13 @@ export function ChartContainer({
             {chart(effectiveChartHeight)}
             {noData}
           </div>
-          <div className={styles["side-legend-container"]} style={{ maxBlockSize: effectiveChartHeight }}>
-            {legend}
+          <div
+            aria-label={"Legend"}
```

Comment:
We cannot hardcode ARIA labels like this, and why do we need it in the first place?

Reply:
Let's remove aria-label from here for now